### PR TITLE
Lever action rifles

### PR DESCRIPTION
**Pull request by Holli-Git**
#### Summary
Content "Adds in new .357, .44, and .22 LR lever action rifles"

#### Purpose of change

I have a list of guns that are commercially common and want to add into the game. This isn't even all of the lever action rifles, as I wanted to keep the line count below 500. 

#### Describe the solution

Adds in a few guns, a few R92s in different calibers and barrel lengths, the Rio Bravo and Henry Golden Boy (both .22 LR rifles), and the Uberti 1873 

#### Describe alternatives you've considered

Just adding in one of each barrel length.

#### Testing

Tested locally, works fine.

### CODE REVIEW

File: data/json/items/gun/38.json [hunk 1]
```diff
@@ -351,5 +351,137 @@
...
+    "copy-from": "rifle_manual",
+    "looks_like": "modular_ar15",
+    "type": "GUN",
+    "name": { "str": ".357 lever-action rifle" },
+    "description": "This lever-action rifle is chambered in the powerful .357 Magnum cartridge, with an 20\" barrel with a tube fed magazine.  Go get 'em cowboy!",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "rossi_r92_357_20",
+        "name": { "str": "Rossi R92 20\"", "str_pl": "Rossi R92s 20\"" },
```
Comment (Hyperseeker) [inline data/json/items/gun/38.json:410]: It may be better to use the double prime symbol `″` to denote inches, since it wouldn't need escaping.

Comment (Holli-Git) [reply data/json/items/gun/38.json:410]: Does using the escape have any flaws with it?

Comment (Hyperseeker) [reply data/json/items/gun/38.json:410]: Using a dedicated (and correct) symbol makes working with the text containing it less error-prone.

Comment (Holli-Git) [reply data/json/items/gun/38.json:410]: As far as I can tell, " is as correct as the double prime. I'm unsure how using the double prime would make it less error-prone

Comment (Hyperseeker) [reply data/json/items/gun/38.json:410]: > Prime marks and quotation marks are not the same thing. <..> a double prime mark is used to show inches or seconds.

— [*Digital Typography: Practical Advice for Getting the Type You Want When You Want It*](https://archive.org/details/digitaltypograph0000gold/page/66)

Comment (Holli-Git) [reply data/json/items/gun/38.json:410]: If you want to replace all the inch notations with double prime, be my guest, but I have no intention or want to change it. It's not going to look any different than a double prime in game anyways

File: data/json/items/gun/22.json [hunk 2]
```diff
@@ -643,5 +643,107 @@
...
       }
     ]
+  },
+  {
+    "id": "rio_bravo",
+    "looks_like": "marlin_9a",
+    "type": "GUN",
+    "reload_noise_volume": 10,
+    "name": { "str": "small game lever-action rifle" },
+    "description": "A .22 Long Rifle lever-action rifle, which holds 15 rounds of ammo.  While quite useful against small-game, its effectiveness isn't as great against the undead.",
```
Comment (Night-Pryanik) [inline data/json/items/gun/22.json:653]:
```suggestion
    "description": "A .22 Long Rifle lever-action rifle, which holds 15 rounds of ammo.  While quite useful against small game, its effectiveness isn't as great against the undead.",
```

File: data/json/items/gun/22.json [hunk 3]
```diff
@@ -643,5 +643,107 @@
...
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "22": 15 } } ],
+    "melee_damage": { "bash": 9 }
+  },
+  {
+    "id": "henry_golden_boy",
+    "looks_like": "marlin_9a",
+    "type": "GUN",
+    "reload_noise_volume": 10,
+    "name": { "str": "small game lever-action rifle" },
+    "description": "A .22 Long Rifle lever-action rifle, which holds 16 rounds of ammo.  While quite useful against small-game, its effectiveness isn't as great against the undead.",
```
Comment (Night-Pryanik) [inline data/json/items/gun/22.json:704]:
```suggestion
    "description": "A .22 Long Rifle lever-action rifle, which holds 16 rounds of ammo.  While quite useful against small game, its effectiveness isn't as great against the undead.",
```

File: data/json/items/gun/38.json [hunk 4]
```diff
@@ -351,5 +351,137 @@
...
     ],
     "melee_damage": { "bash": 6 }
+  },
+  {
+    "id": "uberti_1873",
+    "copy-from": "rifle_manual",
+    "looks_like": "modular_ar15",
+    "type": "GUN",
+    "name": { "str": "reproduction .357 lever-action rifle" },
+    "description": "This fine lever-action rifle is chambered in the powerful .357 Magnum cartridge, with an 20\" barrel with a tube fed magazine.  Go get 'em cowboy!",
```
Comment (Night-Pryanik) [inline data/json/items/gun/38.json:361]:
```suggestion
    "description": "This fine lever-action rifle is chambered in the powerful .357 Magnum cartridge, with an 20\" barrel with a tube-fed magazine.  Go get 'em cowboy!",
```

File: data/json/items/gun/38.json [hunk 5]
```diff
@@ -351,5 +351,137 @@
...
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "357mag": 10, "38": 10 } } ],
+    "melee_damage": { "bash": 12 }
+  },
+  {
+    "id": "rossi_r92_357_20",
+    "copy-from": "rifle_manual",
+    "looks_like": "modular_ar15",
+    "type": "GUN",
+    "name": { "str": ".357 lever-action rifle" },
+    "description": "This lever-action rifle is chambered in the powerful .357 Magnum cartridge, with an 20\" barrel with a tube fed magazine.  Go get 'em cowboy!",
```
Comment (Night-Pryanik) [inline data/json/items/gun/38.json:405]:
```suggestion
    "description": "This lever-action rifle is chambered in the powerful .357 Magnum cartridge, with an 20\" barrel with a tube-fed magazine.  Go get 'em cowboy!",
```

File: data/json/items/gun/38.json [hunk 6]
```diff
@@ -351,5 +351,137 @@
...
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "357mag": 10, "38": 10 } } ],
+    "melee_damage": { "bash": 12 }
+  },
+  {
+    "id": "rossi_r92_357_16",
+    "copy-from": "rifle_manual",
+    "looks_like": "modular_ar15",
+    "type": "GUN",
+    "name": { "str": ".357 lever-action rifle" },
+    "description": "This fine lever-action rifle is chambered in the powerful .357 Magnum cartridge, with an 16\" barrel with a tube fed magazine.  Go get 'em cowboy!",
```
Comment (Night-Pryanik) [inline data/json/items/gun/38.json:449]:
```suggestion
    "description": "This fine lever-action rifle is chambered in the powerful .357 Magnum cartridge, with an 16\" barrel with a tube-fed magazine.  Go get 'em cowboy!",
```

File: data/json/items/gun/44.json [hunk 7]
```diff
@@ -269,5 +269,93 @@
...
     "symbol": "(",
     "color": "dark_gray"
+  },
+  {
+    "id": "rossi_r92_44_16",
+    "copy-from": "rifle_manual",
+    "looks_like": "modular_ar15",
+    "type": "GUN",
+    "name": { "str": ".44 lever-action rifle" },
+    "description": "This lever-action rifle is chambered in the powerful .44 Magnum cartridge, with an 16\" barrel with a tube fed magazine.  Go get 'em cowboy!",
```
Comment (Night-Pryanik) [inline data/json/items/gun/44.json:279]:
```suggestion
    "description": "This lever-action rifle is chambered in the powerful .44 Magnum cartridge, with an 16\" barrel with a tube-fed magazine.  Go get 'em cowboy!",
```

File: data/json/items/gun/44.json [hunk 8]
```diff
@@ -269,5 +269,93 @@
...
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "44": 8 } } ],
+    "melee_damage": { "bash": 12 }
+  },
+  {
+    "id": "rossi_r92_44_20",
+    "copy-from": "rifle_manual",
+    "looks_like": "modular_ar15",
+    "type": "GUN",
+    "name": { "str": ".44 lever-action rifle" },
+    "description": "This lever-action rifle is chambered in the powerful .44 Magnum cartridge, with an 20\" barrel with a tube fed magazine.  Go get 'em cowboy!",
```
Comment (Night-Pryanik) [inline data/json/items/gun/44.json:323]:
```suggestion
    "description": "This lever-action rifle is chambered in the powerful .44 Magnum cartridge, with an 20\" barrel with a tube-fed magazine.  Go get 'em cowboy!",
```

**Pull request merged**